### ISSUE TITLE
Marker Dictionary

### ISSUE THREAD

**Issue by Asif6511** [pal-robotics/aruco_ros#57]
Hi,

Is it possible to use a different Aruco dictionary in this package? I know that now the package can detect markers from Original aruco dictionary. Please suggest a way  to implement the detection on a different Aruco dictionary. Thanks.

**Comment by NalinLuthra** [comment]
Yes, You can easily use different dictionary just need to do some tweaks in the code it self. It will be easy if you would like to tell which dictionary ?

**Comment by Asif6511** [comment]
I would like to use the 7*7 dictionary. I find the marker detector function doesn't require dictionary parameter as supposed to the one in OpenCV arUco documentation. Also, since dictionaries are predefined, I don't find anywhere in the code where a particular dictionary is loaded. Please let me know if you found the snippet to do so. Thanks :)

**Comment by alanprodam** [comment]
Hi,

I have the same doubt, I would like to use the Dictionary: 4x4(50).

**Comment by fhln** [comment]
yah...the same question

**Comment by Sinchiguano** [comment]
same question but nor answer at all, anyone who solved the problem, let me know please.

**Comment by DharminB** [comment]
I think the dictionary used is 5x5. [This function](https://github.com/pal-robotics/aruco_ros/blob/kinetic-devel/aruco/src/aruco/arucofidmarkers.cpp#L316) is pretty much hard coding it. I haven't tried changing it but that might be a good place to start. :man_shrugging:

**Comment by xhqnogit** [comment]
Hello everyone, has anyone solved this problem? I wanted to use the DICT_7X7_100 dictionary, but I couldn't find it. Thank you so much for helping me

**Comment by ErikParkerrr** [comment]
any updates on this?
